irohad/ordering/impl/batches_cache.cpp

```diff
@@ -6,6 +6,7 @@
 #include "ordering/impl/batches_cache.hpp"
 
 #include <fmt/core.h>
+#include <iostream>
```

Comment:
Did you leave this here by accident? There doesn't seem to be any I/O going on.

Reply:
In current version of code this line is not in the file.